src/main/java/io/github/thebusybiscuit/slimefun4/api/ErrorReport.java

```diff
@@ -84,6 +86,10 @@ public ErrorReport(T throwable, SlimefunAddon addon, Consumer<PrintStream> print
     @ParametersAreNonnullByDefault
     public ErrorReport(T throwable, Location l, SlimefunItem item) {
         this(throwable, item.getAddon(), stream -> {
+            Date date = new Date();
+            SimpleDateFormat formatter = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
```

Comment:
This should not be initialised every single error report. Instead make this a global static variable